source/gameengine/Ketsji/KX_LodLevel.h

```diff
@@ -0,0 +1,65 @@
+/*
+* ***** BEGIN GPL LICENSE BLOCK *****
+*
+* This program is free software; you can redistribute it and/or
+* modify it under the terms of the GNU General Public License
+* as published by the Free Software Foundation; either version 2
+* of the License, or (at your option) any later version.
+*
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with this program; if not, write to the Free Software Foundation,
+* Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+*
+* Contributor(s): Porteries Tristan.
+*
+* ***** END GPL LICENSE BLOCK *****
+*/
+
+#include "EXP_Value.h"
+#include "RAS_MeshObject.h"
+
+class KX_LodLevel : public CValue
+{
+	Py_Header
+
+private:
+
+public:
+	KX_LodLevel();
+	virtual ~KX_LodLevel();
+
```

Comment:
@youle: We don't need `m_savedData` struct because here we copied data form blender and after that we modify nothing in the blender data.